static/templates/settings/bot-settings.handlebars

```diff
@@ -41,6 +41,14 @@
                                 {{/if}}
                             {{/each}}
                         </select>
+                    </div>
+                     <div class="input-group" id="service_name_list">
```

Comment:
Look carefully. This line and the closing tag above this don't line up. These should both be at 22 spaces indent. This is the most likely cause of your test failures.